package.json

```diff
@@ -60,6 +62,7 @@
   "files": [
     "*.js"
   ],
+  "types": "index.d.ts",
```

Comment:
`index.d.ts` is not included in `files`, so it's not part of the npm package:

https://unpkg.com/@cypress/webpack-preprocessor/

Reply:
D'oh! I'll get that fixed and a new release out right away.

Reply:
Fixed in [`5.1.1`](https://github.com/cypress-io/cypress-webpack-preprocessor/releases/tag/v5.1.1)

Reply:
I could have sworn the types field was included. Main is, but types aren't I guess :(
https://docs.npmjs.com/files/package.json